docs/source/docker_only.rst

```diff
@@ -0,0 +1,129 @@
+
+Enter the experiment directory:
+
+.. code-block:: shell
+
+   cd /path/to/experiment-directory
+
+Create a file named `Dockerfile` with these contents (replace image name in the `FROM` directive to match the dallinger version you depend on in your `constraints.txt` file):
+
+.. code-block::
+
+    # syntax = docker/dockerfile:1.2
+    FROM ghcr.io/dallinger/dallinger:9.1.0a1
+
+    RUN mkdir /experiment
+    COPY requirements.txt /experiment/constraints.txt
+
+    WORKDIR /experiment
+
+    RUN python3 -m pip install -e /dallinger[docker]
+
+    # If a dependency needs the ssh client and git, install them
+    RUN grep git+ constraints.txt && \
+        apt-get update && \
+        apt-get install -y openssh-client git && \
+        rm -rf /var/lib/apt/lists || true
+
+    # We rely on the already installed dallinger: the docker image tag has been chosen
+    # based on the contents of this file. This makes sure dallinger stays installed from
+    # /dallinger, and that it doesn't waste space with two copies in two different layers.
+    RUN mkdir ~/.ssh && echo "Host *\n    StrictHostKeyChecking no" >> ~/.ssh/config
+    RUN --mount=type=ssh grep -v ^dallinger constraints.txt > /tmp/requirements_no_dallinger.txt && \
+        python3 -m pip install -r /tmp/requirements_no_dallinger.txt
+    ENV PORT=5000
+    CMD dallinger_heroku_web
+
+Build a docker image for the experiment using Buildkit:
+
+.. code-block:: shell
+    EXPERIMENT_IMAGE=my-experiment
+    DOCKER_BUILDKIT=1 docker build . -t ${EXPERIMENT_IMAGE}
+
+Start the development server with docker:
+
+.. code-block:: shell
+
+    docker run --name dallinger --rm -ti -u $(id -u ${USER}):$(id -g ${USER}) -v ${PWD}:/experiment --network dallinger -p 5000:5000 -e FlASK_OPTIONS='-h 0.0.0.0' -e REDIS_URL=redis://dallinger_redis:6379 -e DATABASE_URL=postgresql://dallinger:dallinger@dallinger_postgres/dallinger ${EXPERIMENT_IMAGE} dallinger develop debug
+
+You can now access the running dallinger instance on http://localhost:5000/dashboard
+The admin password can be found in the develop `config.txt` file:
+
+.. code-block:: shell
+
+    grep dashboard_password ./develop/config.txt
+
+
+Deploy the experiment image using ssh
+=====================================
+
+We're going to use variations of the same command, so we create an alias for convenience.
+
+.. code-block:: shell
+
+    alias docker-dallinger='docker run --rm -ti -v ~/.ssh:/home/.ssh -v /etc/passwd:/etc/passwd -v /etc/shadow:/etc/shadow -v ~/.local/share/dallinger/:/home/.local/share/dallinger/ -e HOME=/home -e DALLINGER_NO_EGG_BUILD=1 -u $(id -u ${USER}):$(id -g ${USER}) -v /var/run/docker.sock:/var/run/docker.sock -v ${PWD}:/experiment  ${EXPERIMENT_IMAGE} dallinger'
+
+
+Then we can use the alias to run dallinger inside a container:
+
+.. code-block:: shell
+
+    docker-dallinger docker-ssh servers list
+
+Create a remote server with
+
+.. code-block:: shell
+
+    docker-dallinger docker-ssh servers add
```

Comment:
I remember having a problem with the servers list having more than one entry. Maybe this is related?

Reply:
There was a bug regarding that (one extra comma that should not have been there). It's fixed in this PR.